iree/tools/CMakeLists.txt

```diff
@@ -217,6 +217,10 @@ if(${IREE_BUILD_COMPILER})
     HOSTONLY
   )
 
+if(${IREE_BUILD_TORCH_MLIR_DIALECTS})
+  set(optional_tmtensor_conversion_lib iree::compiler::InputConversion::TMTensor)
```

Comment:
nit: we typically use `ALL_CAPS` for variable names, and sometimes prefix them with an underscore (`_ALL_CAPS`) for "local" variables, so this could be `_OPTIONAL_TMTENSOR_CONVERSION_LIB`

also please indent this block to match the surrounding code